lib/simp/cli/puppetfile/skeleton.rb

```diff
@@ -0,0 +1,58 @@
+require 'simp/cli'
+module Simp::Cli::Puppetfile
+  # Provides a skeleton Puppetfile that can include a local Puppetfile.simp
+  class Skeleton
+    # @return [String] all modules' data as a Puppetfile
+    def self.to_puppetfile
+      <<-PUPPETFILE.gsub(/ {8}/,'')
+        # ==============================================================================
+        # SIMP Puppet modules
+        # ------------------------------------------------------------------------------
+        # The line below enables this Puppetfile to deploy all of SIMP's modules from a
+        # neighboring `Puppetfile.simp` file.
+        #
+        # If you install SIMP modules locally from RPMs, you can generate a current
+        # `Puppetfile.simp` at any time by running the command:
+        #
+        #     simp puppetfile generate > Puppetfile.simp
+        #
+        # You can regenerate a clean copy of this Puppetfile at any time by running:
+        #
+        #     simp puppetfile generate --skeleton > Puppetfile
+        #
+        # ------------------------------------------------------------------------------
+        instance_eval(File.read("Puppetfile.simp"))
+
+
+        # ==============================================================================
+        # Your site's Puppet modules
+        # ------------------------------------------------------------------------------
+        # Add your own Puppet modules here
+
+
+        # ------------------------------------------------------------------------------
```

Comment:
I don't really get that this is supposed to be with the previous section (is it?)

Reply:
Nope, it's just a separate comment marking where you should put your own stuff (as opposed to the SIMP section above).